include/EffectSelectDialog.h

```diff
@@ -65,10 +65,16 @@ class DualColumnFilterProxyModel : public QSortFilterProxyModel
 		QString name = sourceModel()->data(nameIndex, Qt::DisplayRole).toString();
 		QString type = sourceModel()->data(typeIndex, Qt::DisplayRole).toString();
 
-		QRegExp nameRegExp(filterRegExp());
-		nameRegExp.setCaseSensitivity(Qt::CaseInsensitive);
+		// TODO: cleanup once we drop Qt5 support 
+		#if (QT_VERSION >= 0x051200)
```

Comment:
Please don't use a magic number here, go back to `QT_VERSION_CHECK`. If it's not working for some reason include `QtVersionChecks`.

Reply:
Done.